packages/isaac-typescript-definitions-repentogon/src/types/classes/item-config/ItemConfig.d.ts

```diff
@@ -14,7 +13,7 @@ declare global {
       tags: ItemConfigTag | BitFlags<ItemConfigTag>,
     ) => ItemConfigItem[];
 
-    /** Returns whether the trinket is valid. */
-    IsValidTrinket: (trinketType: TrinketType) => boolean;
+    // IsValidTrinket seems to be commented out for some reason in the API yet is documented as
```

Comment:
in situations like this, I typically include upstream links.
Here, it would be nice to include a link to the code on github, and then a link to the API docs, so that future  readers can easily just click on the links to see if the bug still exists.